js/store/actions.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
```

Comment:
Strict mode is activated by default on JS modules, so when using the `import` this directive is not required anymore.